packages/yoga/src/Theme/Provider/web/GlobalStyle.jsx

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyle = createGlobalStyle`
+  ${({
+    theme: {
+      yoga: { baseFontFamily },
+    },
+  }) => `
+    @import url(https://fonts.googleapis.com/css?family=${baseFontFamily}:400,600,700,400i,600i,700i&display=swap);
```

Comment:
This is a really cool solution to load the `font-family`!

BUT, we may want to receive this uri as a parameter that defaults to the google api. We, as a company, should use our CDN to serve these static files, which will be much faster than the Google API.

(We can see this when loading the Yoga documentation and the font flashes when it is loaded [the font was not downloaded yet])

We should also see how should we expose these fonts through the CDN.

Reply:
Glad you liked, @tcK1 !

The link you notice us isn't a static file. The Google [Fonts API provides](https://fonts.google.com/about) us an automatically user-agent identification, sending the smallest possible file, caching fonts without compromising user's privacy or security. Also, the cache is made cross-site, making the needs of loading the font just once, with any website and the fonts API is 58% faster than our CDN (measured just for font loading).

The blinking font effect is a UX decision and it could be changed using the [font-display](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display) CSS property which, in this case, is hardcoded as `swap`.